pwndbg/gdblib/vmmap.py

```diff
@@ -112,9 +123,6 @@ def get() -> Tuple[pwndbg.lib.memory.Page, ...]:
         elif kernel_vmmap == "monitor":
             pages.extend(kernel_vmmap_via_monitor_info_mem())
 
-    if not pages and is_corefile():
-        pages.extend(coredump_maps())
-
     # TODO/FIXME: Do we still need it after coredump_maps()?
     # Add tests for other cases and see if this is needed e.g. for QEMU user
     # if not, remove the code below & cleanup other parts of Pwndbg codebase
```

Comment:
I guess if this really isn't needed, just removing it would fix the recursion issues as well?

Reply:
I rephrased this. I probably need to test this edge case a bit more.